Makefile

```diff
@@ -139,8 +150,11 @@ prod-push:
 ###################################
 
 # Create YAML file for deployment into CI
-ci-yaml: | $(KUSTOMIZE)
-	CAPV_MANAGER_IMAGE=${CI_IMG}:${VERSION} cmd/clusterctl/examples/vsphere/generate-yaml.sh
+ci-yaml: clusterctl-tools
```

Comment:
Hi @andrewsykim,

This should be an order-only dependency. Please change it to:

```make
ci-yaml: | clusterctl-tools
```